python/feldera/rest/config.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+
+class Config:
+    """
+    Client's credentials and configuration parameters
+    """
+
+    def __init__(
+            self,
+            url: str,
+            api_key: Optional[str] = None,
+            version: Optional[str] = None,
+            timeout: Optional[str] = None,

```

Comment:
What format does the timeout take?

Reply:
Actually timeout should be a `float` not `str`, it is consistent to `requests.request`'s `timeout` parameter.